ramls/po_number.raml

```diff
@@ -0,0 +1,37 @@
+#%RAML 1.0
+title: "Orders Storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v1
+
+documentation:
+  - title: Purchase Order Number
+    content: <b>API used to manage PO number.</b>
+
+types:
+  po_number: !include acq-models/mod-orders-storage/schemas/po_number.json
+  UUID:
+    type: string
+    pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+    language: !include raml-util/traits/language.raml
+
+/orders-storage/po_number:
+  get:
+    is: [language]
+    description: Get generated purchase order number
+    responses:
+      200:
+        body:
+          application/json:
+            type: po_number
+      400:
```

Comment:
I can't think of any situations where this would happen... Did you have a particular scenario in mind?

Reply:
I've found that code 400 appears only when po_number sequence is absent or it has restricted access.  Sure, its almost impossible scenario. Do you think code 400 should be removed from raml?

Reply:
Yes, I think it should probably be removed.  Given that this endpoint doesn't take any input from the client, it's hard to blame anything that goes wrong on the client (e.g. 400).  In the case where the sequence is missing or can't be accessed, that should result in a 500.